examples/oneview_server_profile.yml

```diff
@@ -141,6 +141,17 @@
 
     - debug: var=server_profile
 
+    - name: Unassign Server Hardware from Server Profile
+      oneview_server_profile:
+        config: "{{ config }}"
+        # This is required for unassigning a SH, or creating a SP and not auto-assigning a SH
+        auto_assign_server_hardware: False
+        data:
+          name: "{{ inventory_hostname }}"
+          # Specify a blank serverHardwareName or serverHardwareUri when auto_assign_server_hardware is False to unassign a SH
+          serverHardwareName:
```

Comment:
It wouldn't be better use use `None` instead of                    ?

Reply:
That is exactly `None`, writing out `None` or `none` would be understood as a string